module/modmanager/manager.go

```diff
@@ -700,7 +700,7 @@ func (m *module) registerResources(mgr modmaninterface.ModuleManager, logger gol
 		switch {
 		case api.API.IsComponent():
 			for _, model := range models {
-				logger.Debugw("registering component from module", "module", m.name, "API", api.API, "model", model)
+				logger.Infow("registering component from module", "module", m.name, "API", api.API, "model", model)
```

Comment:
Changed this as part of another small ticket in rdk: https://viam.atlassian.net/browse/APP-2700?atlOrigin=eyJpIjoiYTNlZDUyMzllYzc3NDU0YjgxYTUxNjdiNjc5Y2I5YTYiLCJwIjoiaiJ9